src/EditorFeatures/Test2/FindReferences/FindReferencesTests.OrdinaryMethodSymbols.vb

```diff
@@ -3254,6 +3254,51 @@ End Class
         }
         </Document>
     </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestOrdinaryMethodWithMissingReferences_CSharp(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="C#" CommonReferences="false">
+        <Document>
+        class C
+        {
+            // string will be an error type because we have no actual references.
+            private void {|Definition:Goo|}(string s) { }
+
+            void Bar()
+            {
+                [|Go$$o|]("");
+                [|Goo|](s);
+            }
+        }
+        </Document>
+    </Project>
+</Workspace>
+            Await TestAPIAndFeature(input, kind, host)
+        End Function
+
+        <WpfTheory, CombinatorialData, Trait(Traits.Feature, Traits.Features.FindReferences)>
+        Public Async Function TestOrdinaryMethodWithMissingReferences_VB(kind As TestKind, host As TestHost) As Task
+            Dim input =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferences="false">
+        <Document>
+        class C
+            ' string will be an error type because we have no actual references.
```

Comment:
Is there some risk here that if somebody breaks the meaning of `CommonReferences="false"` this test can still pass and we lose coverage? Don't we have SymbolKey tests directly?

Reply:
good point.  wil make a direct symbolkey test.

Reply:
done.